src/coaches/coaches.controller.ts

```
@@ -0,0 +1,23 @@
+import { CoachesService } from './coaches.service';
+
+@Controller('coaches')
+export class CoachesController {
+    constructor(private readonly coachesService: CoachesService) {}
+    
+    // @UseInterceptors(LoggingInterceptor)
+    @Get()
+    getAll(){
+        return this.coachesService.findAll();
+    }
+    @Get('id/:id')
```

ESLint

[error] Delete `··`

(prettier/prettier)